pytorch_lightning/loops/fit_loop.py

```diff
@@ -185,23 +189,6 @@ def on_run_start(self) -> None:  # type: ignore[override]
         # reset train dataloader and val dataloader
         self.trainer.reset_train_val_dataloaders(self.trainer.lightning_module)
 
-        ft_enabled = _FaultTolerantMode.detect_current_mode().is_enabled
```

Comment:
quick ques: we are not raising this warning anymore even if ckpt is mid epoch but no fault-tolerant is enabled?

Reply:
so is it fault-tolerant by default?

Reply:
The loop progress restoration yes.

The dataloading, rng, etc no.

Reply:
if dataloader isn't reset and is going to iterate over it again completely, to me it looks more like a fresh epoch start, even though the progress is intact from the previous run.

Reply:
I looked further into this. I added back this warning and it's tests. But I had to move it one loop down.